packages/blockchain-wallet-v4-frontend/src/data/auth/selectors.ts

```diff
@@ -11,3 +15,9 @@ export const getMobileLoginStarted = path(['auth', 'mobileLoginStarted'])
 export const getRegisterEmail = path(['auth', 'registerEmail'])
 export const getMetadataRestore = path(['auth', 'metadataRestore'])
 export const getKycResetStatus = path(['auth', 'kycReset'])
+
+export const getMagicLinkData = (state: RootState): WalletDataFromMagicLink | null =>
+  state.auth.magicLinkData
+export const getWalletData = (state) => getMagicLinkData(state)?.wallet
+export const getNabuId = (state) => getWalletData(state)?.nabu?.userId
+export const getRecoveryToken = (state) => getWalletData(state)?.nabu?.recoveryToken
```

Comment:
consider just having `getMagicLinkData` or at least add suffix last 3 to say `FromMagicLink`